src/Adapter/Configuration.php

```diff
@@ -332,9 +332,9 @@ private function getShopGroupId(?ShopConstraint $shopConstraint): ?int
     /**
      * @param ShopConstraint|null $shopConstraint
      *
-     * @return int|null
+     * @return bool
      */
-    private function isStrict(?ShopConstraint $shopConstraint): ?int
+    private function isStrict(?ShopConstraint $shopConstraint): bool
```

Comment:
Not a BC break it's a private method, the incorrect return type was copied from the previous method